tekton-catalog/condition/condition.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Condition
+metadata:
+  name: super-condition
+spec:
+  check:
+    args:
```

Comment:
The script looks good to me now. Can you also update the static template to follow the same design?